infrastructure/main.tf

```diff
@@ -141,6 +146,7 @@ module "case-service" {
     FPL_LOCAL_AUTHORITY_CODE_TO_HMCTS_COURT_MAPPING     = "${data.azurerm_key_vault_secret.local_authority_code_to_hmcts_court_mapping.value}"
     FPL_LOCAL_AUTHORITY_CODE_TO_CAFCASS_MAPPING         = "${data.azurerm_key_vault_secret.local_authority_code_to_cafcass_mapping.value}"
     FPL_LOCAL_AUTHORITY_CODE_TO_SHARED_INBOX_MAPPING    = "${data.azurerm_key_vault_secret.local_authority_code_to_shared_inbox_mapping.value}"
+    FPL_GENERAL_FPLA_INBOX_MAPPING                      = "${data.azurerm_key_vault_secret.general_fpla_inbox_mapping.value}"
     NOTIFY_API_KEY                                      = "${data.azurerm_key_vault_secret.notify_api_key.value}"
```

Comment:
Why the FPL and FPLA? General really tells me nothing about what this mapping is.

Reply:
changed it to `FPL_PUBLIC_LAW_EMAIL_MAPPING`

Reply:
Hmm, FPL is (Family Public Law) don't think we need the second PUBLIC_LAW here. Maybe just `PUBLIC_LAW_EMAIL_MAPPING` or `FPL_EMAIL_MAPPING`

Reply:
changed to `FPL_EMAIL_MAPPING`

Reply:
My only question is do you need to change anything else now that this name has changed?

Reply:
It should match with application.yaml which is `FPL_PUBLIC_LAW_EMAIL_MAPPING`. I need to revert it back. Or shall we call in default email??

Reply:
As discussed @lewisbirks @Gogetter, I have changed this to `FPL_DEFAULT_EMAIL_MAPPING` and also done changes in other config and classes for the same.